topics/go/language/embedding/exercises/exercise1/exercise1.go

```diff
@@ -53,12 +53,12 @@ func main() {
 	}
 
 	// Display what we are trying to find.
-	fmt.Println("Searching For:", needle)
+	fmt.Printf("Searching For: %s\n", needle)
```

Comment:
No behavior change here. Revert to Println

Reply:
Agree